tracer/test/test-applications/azure-functions/Samples.AzureFunctions.V3InProcess/AllTriggers.cs

```diff
@@ -64,13 +66,20 @@ public async Task ExitApp([TimerTrigger(AtMidnightOnFirstJan, RunOnStartup = tru
             return new OkObjectResult("Attempting triggers.");
         }
 
-        private async Task<string> CallFunctionHttp(string path, ILogger logger)
+        private async Task<HttpResponseMessage> CallFunctionHttp(string path, ILogger logger)
```

Comment:
Did you mean to include this change in this PR? It seems unrelated

Reply:
This PR showed an issue that only came up in the azure function tests, which are quite difficult to debug. This was an attempt to add extra logging, but the changes didn't provide that much extra info in the end, I think it's probably worth keeping them, but don't have a strong view on this.